site/src/components/WorkspaceDeletion/DeletionBadge.tsx

```diff
@@ -0,0 +1,31 @@
+import { Workspace } from "api/typesGenerated"
+import { displayImpendingDeletion } from "./utils"
+import { useDashboard } from "components/Dashboard/DashboardProvider"
+import { Pill } from "components/Pill/Pill"
+import ErrorIcon from "@mui/icons-material/ErrorOutline"
+
+export const DeletionBadge = ({
```

Comment:
nit: Since we are using the copy "Impending deletion" should we not call this `ImpendingDeletionBadge`?

Reply:
Sure! I can adjust the names